src/main/deploy/swerve/modules/backleft.json

```diff
@@ -0,0 +1,27 @@
+{
+  "location": {
+    "front": 0,
+    "left": 0
+  },
+  "absoluteEncoderOffset": 0,
+  "drive": {
+    "type": "sparkflex",
+    "id": 0,
+    "canbus": null
+  },
+  "angle": {
+    "type": "sparkflex",
+    "id": 0,
+    "canbus": null
+  },
+  "encoder": {
+    "type": "none",
```

Comment:
Absolute encoder type can be `talonsrx_pwm` for all modules. We'll pull in a pr'd version before we start testing that adds this option, https://github.com/BroncBotz3481/YAGSL-Example/pull/208